code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -368,3 +368,114 @@ obj/item/weapon/storage/box/syndie_kit/cheaptide
 	..()
 	new /obj/item/device/telepad_beacon(src)
 	new /obj/item/weapon/rcs/salvage/syndicate(src)
+
+
+//Elite Syndicate Bundles
+//for all of the team bundles
+
+/obj/item/weapon/storage/box/syndie_kit/sniper
+	name = "Sniper"
+
+/obj/item/weapon/storage/box/syndie_kit/sniper/New()
+	..()
+	new /obj/item/device/radio/headset/headset_earmuffs/syndie(src)
+	new /obj/item/weapon/gun/projectile/hecate(src)
+	new /obj/item/clothing/accessory/storage/webbing(src)
+	new /obj/item/ammo_storage/box/BMG50(src)
+	new /obj/item/ammo_storage/box/BMG50(src)
+	new /obj/item/ammo_storage/box/BMG50(src)
+	new /obj/item/clothing/glasses/thermal/syndi(src)
+	return
```

Comment:
no trailing returns, ditto for ALL of these

Reply:
done, compiled, and tested